### DISCUSSION TITLE
Kerberos libs in master

### DISCUSSION THREAD

**Discussion by Selllavi** [cloudera/hue#2503] [Questions] [answered]
Hello, why  "krb5-config", "krb5-user" libs which were added there:

> https://github.com/cloudera/hue/commit/60264e58932c4c84e6d43fb1b838e7cfb1c5aaa9

when migration to python3 was done 

> https://github.com/cloudera/hue/commit/003855714392b8ad5781c236d0e71dedb19198df#diff-177d829dd727371c45d25aaf1c76baad9c974dc4b0da342fde25fcdaf87c8b1e

were left only for Python 2 Dockerfile version? 
Is it possible to add them into Dockerfile to Python 3 as well?

**Comment by erplus** [comment]
I agree. In the latest release image (currently [4.10.0](https://hub.docker.com/layers/gethue/hue/4.10.0/images/sha256-5702b2c37ff94129cab6b27281047668872960920363c2eb88a93f7b1ffb68a1?context=explore)) the only reason i have to rebuild this image is getting install "krb5-user" package to get kinit.

In my  opinion "krb5-config" and "krb5-user" should be added to the [Dockerfile for python3 version](https://github.com/cloudera/hue/blob/master/tools/docker/hue/Dockerfile)

**Comment by Harshg999** [comment, marked as answer]
Added the packages via https://github.com/cloudera/hue/pull/2797.
